metadata-ingestion/src/datahub/utilities/mapping.py

```diff
@@ -160,7 +178,7 @@ def process(self, raw_props: Mapping[str, Any]) -> Dict[str, Any]:
                                 operation_type, set()
```

Comment:
let's add a special case for ownership, above the `if isinstance(operation, (str, list)):` line

operation_map[owner operation] should be a list, not a set.

Reply:
done